docs/index.rst

```diff
@@ -0,0 +1,59 @@
+================
+spin1_beam_model
+================
+
+This is the documentation of **spin1_beam_model**.
+
+.. note::
+
+    This is the main page of your project's `Sphinx`_ documentation.
```

Comment:
Should we edit this text at all? I'm not familiar with doing formal documentation like this, so forgive the naive question...